packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -359,62 +360,51 @@ export class AnchoredRegion extends FoundationElement {
     @observable
     public initialLayoutComplete: boolean = false;
 
+    /**
+     * indicates that region contents are now loaded and visible
+     *
+     * @internal
+     */
+    @observable
+    public regionVisible: boolean = false;
```

Comment:
```suggestion
    public visible: boolean = false;
```

Reply:
"visible" is already a property on html element with a different meaning, so I don't think having an identical name is a good idea.

Reply:
Do you have a link to this? I’m not seeing this on the generic element or HTML element properties.

Reply:
And note that is labelled as "@internal"

Reply:
Sorry, I was apparently confused about the existence of a visible prop.

Reply:
I think "contentRendered" might be a more accurate name for this.